### PR TITLE
add software update available for bridge

### PR DESCRIPTION
**Pull request by starkillerOG**
I would like to add a prompt to HomeAssistant (warning in log) for user to update their Philips Hue bridge if their is an update available.

### CODE REVIEW

File: aiohue/config.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ def swversion(self):
         """Software version of the bridge."""
         return self.raw['swversion']
 
+    @property
+    def swupdatebridgestate(self):
```
Comment (balloob) [inline aiohue/config.py]: let's add some underscores for readability.

Comment (balloob) [reply aiohue/config.py]: Also make sure it's `swupdate2`, we should not make up our own names.

**Comment by starkillerOG** [comment]
@balloob, I just renamed it.

**Comment by balloob** [comment]
Release as 1.9.1

**Comment by starkillerOG** [comment]
Thank you for releasing a new version!

**Pull request merged**